### DISCUSSION TITLE
Why does average processing time in summary and upstream section differ?

### DISCUSSION THREAD

**Discussion by webmihir** [AdguardTeam/AdGuardHome#6219] [General]
My AGH has a single upstream server setup, as you can see in the [screenshot](https://imgur.com/a/Bssej0f) (sensitive parts redacted from screenshot). Why does the summary section show average processing time of 11ms, whereas the upstream processing time show as 59ms?

**Comment by timkgh** [comment]
AGH caches DNS entries so you will see the effects of caching in the avg processing time (lower than upstream processing time is a good thing).

**Reply by webmihir** [reply]
Ah ok, that makes sense. So the lower number is because requests are not sent upstream as they were in the cache. Thanks!
